src/app/components/navigation/Footer.tsx

```diff
@@ -4,40 +4,50 @@ import { ListGroupFooter } from "../elements/list-groups/ListGroupFooter";
 import { ListGroupSocial } from "../elements/list-groups/ListGroupSocial";
 import { ListGroupFooterBottom } from "../elements/list-groups/ListGroupFooterBottom";
 import { Link } from "react-router-dom";
+import { ExternalLink } from "../elements/ExternalLink";
 
 export const Footer = () => (
   <footer>
     <div className="footerTop d-print-none">
       <Container>
-        <Row className="px-3 px-sm-0 pb-3 pb-md-4">
-          <Col md="4" lg="3" className="pt-5 logo-col">
-            <div className="d-flex flex-row">
-              <Link to="/">
-                <img src="/assets/logo-mustard.svg" className="footerLogo" alt="Isaac Computer Science homepage" />
-              </Link>
-            </div>
+        <Row className="px-3 px-sm-0 pb-3 pb-md-4 footer-content">
+          <Col xs="12" lg="3" className="pt-5 logo-col">
+            <Link to="/">
+              <img
+                src="/assets/logo_footer.svg"
+                className="footerLogo d-block w-100"
+                alt="Isaac Computer Science homepage"
+              />
+            </Link>
             <div className="footer-links logo-text pt-3">
               <p>
                 Isaac Computer Science is part of the
-                <Link to="/teachcomputing">National Centre for Computing Education.</Link>
+                <Link to="/teachcomputing" style={{ color: "white", textDecoration: "none" }}>
+                  National Centre for Computing Education.
+                </Link>
               </p>
             </div>
           </Col>
-          <Col md={{ size: 7, offset: 1 }} lg={{ size: 5, offset: 1 }} className="pt-5 mt-4 mt-md-0">
+          <Col xs="12" lg="9" xl={{ size: 8, offset: 1 }} className="pt-5 links-col">
             <ListGroupFooter />
-          </Col>
-          <Col md="5" lg="3" className="pt-5 mt-4 mt-lg-0">
             <ListGroupSocial />
           </Col>
         </Row>
       </Container>
     </div>
     <div className="footerBottom">
       <Container>
-        <Row className="pt-3 px-3 px-sm-0 pb-3">
+        <Row className="pt-3 px-3 px-sm-0">
           <ListGroupFooterBottom />
         </Row>
       </Container>
+      <div>
```

Comment:
I'd either say this div is redundant, or we should explicitly set it to be full width, which then makes the below easier. That's because you'd rather use flex these days over float, and by having this parent full width, we could simply add a couple of classes to do the following: 

```js
<div className="w-100 d-flex justify-content-end"> 
  <ExternalLink href="https://www.stem.org.uk/">
    <img src="/assets/logos/stem_footer.svg" alt="STEM Learning" className="logo-mr" height="100px" />
  </ExternalLink>
</div>
```

this might not work exactly, but that's the gist :)